components/Banner/examples/supplemental/MDCBannerView.m

```diff
@@ -0,0 +1,119 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCBannerView.h"
+
+#import "MDCFlatButton.h"
+#import "MaterialColorScheme.h"
+#import "MaterialTypography.h"
+
+@interface MDCBannerView ()
+
+@property(nonatomic, readwrite, weak) UILabel *textLabel;
+@property(nonatomic, readwrite, strong) NSMutableArray<__kindof UIButton *> *buttons;
+
+@end
+
+@implementation MDCBannerView
+
+- (instancetype)initWithFrame:(CGRect)frame {
+  self = [super initWithFrame:frame];
+  if (self) {
+    [self commonInit];
+  }
+  return self;
+}
+
+- (instancetype)initWithCoder:(NSCoder *)aDecoder {
+  self = [super initWithCoder:aDecoder];
+  if (self) {
+    [self commonInit];
+  }
+  return self;
+}
+
+- (void)commonInit {
```

Comment:
In order to avoid name collision, this needs to be named commonBannerViewInit

Reply:
Renamed in b1f4f19db88e819349821db4f408391ce7bc54d2.